drape_frontend/rule_drawer.cpp

```diff
@@ -322,7 +322,7 @@ void RuleDrawer::ProcessAreaAndPointStyle(FeatureType & f, Stylist const & s, TI
     apply.ProcessAreaRule(s.m_areaRule, areaDepth, false);
 
   /// @todo Can we put this check in the beginning of this function?
-  if (!IsDiscardCustomFeature(f.GetID()))
+  if (applyPointStyle && !IsDiscardCustomFeature(f.GetID()))
```

Comment:
1. [drape] Fixed possible uninitialized center point for area Feature. => It's not "possible", it's definitely uninitialized :)
2. I confirm that this fix works.
3. The code of this function looks strange. Why ApplyAreaFeature is used even if applyPointStyle is true, for example?

Reply:
Area Feature process area (hatching, filling) and point (symbol, text) styles.